packages/java-parser/package.json

```diff
@@ -12,5 +12,14 @@
     "test": "npm-run-all clone-samples test:execute",
     "test:execute": "mocha \"test/**/*-spec.js\"",
     "clone-samples": "node ./scripts/clone-samples"
+  },
+  "devDependencies": {
```

Comment:
I am not sure we want these common dev deps at the package level as it would create
duplicates and **multiple** versions of dev tools co-existing at the same time on the same mono repo.

How do other mono repos handle this issue?